docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -244,7 +244,7 @@ Instead, the first argument to [can-construct.extend] is the name of the constru
 If you use StealJS, you’ll need to install [steal-stache] to load your templates:
 
 ```
-npm install steal-stache --save
+npm install steal-stache@3 --save
```

Comment:
Yeah, this is not required right now, but I think we should typically version our install commands so new and old guides install the correct version.